core/utils.py

```diff
@@ -160,36 +160,42 @@ def create_embed(
         *,
         error: bool = False,
         author: Union[discord.Member, discord.User] = None,
-        **kwargs,
+        title: Union[str, discord.Embed.Empty] = discord.Embed.Empty,
+        description: Union[str, discord.Embed.Empty] = discord.Embed.Empty,
+        footer: Optional[str] = None,
+        color: Optional[int, discord.Colour] = None,
```

Comment:
This line would throw `TypeError: Optional[t] requires a single type.` There has to be an union inside.
```py
Optional[Union[int, discord.Colour]]
```

Reply:
Thanks, haven't had the opportunity to look too in depth in python types annotations, just learnt some things!

Reply:
And we thank you for choosing to contribute to this project of ours!

It might not look like it but we appreciate it very much.